src/operator-docs/transformation/groupBy.ts

```diff
@@ -1,6 +1,123 @@
-import { OperatorDoc } from '../operator.model';
+import { OperatorDoc } from "../operator.model";
 
 export const groupBy: OperatorDoc = {
-  'name': 'groupBy',
-  'operatorType': 'transformation'
+  name: "groupBy",
+  operatorType: "transformation",
+  signature: `
+    public groupBy(keySelector: (value: T) => K,
+                   elementSelector?: ((value: T) => R) | void,
+                   durationSelector?: (grouped: GroupedObservable<K, R>) => Observable<any>,
+                   subjectSelector?: () => Subject<R>)
+          : Observable<any>): OperatorFunction<T, GroupedObservable<K, R>>`,
+  parameters: [
+    {
+      name: "keySelector",
+      type: "(value: T) => K",
+      attribute: "",
+      description: `A function that extracts the key used for grouping for each item.`
+    },
+    {
+      name: "elementSelector",
+      type: "((value: T) => R) | void",
+      attribute: "optional",
+      description: `A function that extracts the emitted element for each item. Default is identity function.`
+    },
+    {
+      name: "durationSelector",
+      type: "(grouped: GroupedObservable<K, R>) => Observable<any>",
+      attribute: "optional",
+      description: `A function that returns an Observable to determine how long each group should exist.`
+    },
+    {
+      name: "subjectSelector",
+      type: "() => Subject<R>",
+      attribute: "optional",
+      description: ``
+    }
+  ],
+  marbleUrl: "http://reactivex.io/rxjs/img/groupBy.png",
+  shortDescription: {
+    description: `
+      Group, according to a specified key, elements from items emitted by an Observable,
+      and emit these grouped items as GroupedObservables, one GroupedObservable per group.
```

Comment:
Is there something we can link to here for `GroupedObservable` that provides a better explanation? Since it is mentioned several times in the explanation I think it would be easy to get lost within an understanding of this concept.

Reply:
Sure, I added some explanation about GroupedObservable.